charts/tekton-apps/templates/_snippets.tpl

```diff
@@ -215,3 +214,10 @@ the way you use it:
   value: {{ .default }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Get namespace and return component `destinationNamespace` if it exists, otherwise return project's `namespace`.
```

Comment:
Pls add here example how to use this snippet with passing `component_name` in the comments

Reply:
Added https://github.com/saritasa-nest/saritasa-devops-helm-charts/commit/69191ae84f8793e265419d7640ed62d0c60e77bb